docs/submitted-data.md

```diff
@@ -2,4 +2,103 @@
 id: submitted-data
 title: Viewing Submitted Data
 ---
-Coming Soon
+Program members can track the progress of their program’s clinical and molecular data submission and download analysis files from the **Program Dashboard**. 
+
+When a program member logs in to the ARGO Data Platform, and navigates to the **Program Services** area in the top menu, they will land on the program dashboard.  If a member belongs to multiple programs, each program has its own dashboard and submission sections.
+
+Program progress is summarized within the following dashboard "cards": 
+
+## Program Statistics 
+
+This card aggregates the following program statistics:
+![Dashbard Program Statistics](/assets/submission/dashboard-stats.png)  
+
+#### Registered Donors
+Once samples are registered, each corresponding donor will be counted as a percentage out of the number of committed program donors.
+
+#### % Donors with all Core Clinical Data
+Once all core clinical fields and files have been submitted for a donor, they are considered **"clinically complete"**. 
+> **A donor must be clinically complete before any of their molecular analysis files are released to the program members.**
+
+Check out [Clinical Data Validation Rules](clinical-data-validation-rules#clinical-data-completeness) for more information on how clinical completeness is calculated.
+
+#### % Donors with Tumour & Normal
+*Coming Soon:* Once a donor has a tumour and normal raw reed pair [submitted through Song](submitting-molecular-data), they will be counted towards this percentage.
+
+#### Donors in Molecular Data Processing
+*Coming Soon:* This is the number of donors that currently have samples being processed in the [analysis pipeline](dna-pipeline).
+
+#### Files to QC
+*Coming Soon:* Once an analysis workflow is completed (and the corresponding donor is clinically complete), the analysis files will be available to program members first. These files can be downloaded using the file manifest, and will be available for an embargo period before being released to the public via the File Repository (note: all molecular files are controlled and will be available for the public to download with [DACO approval](data-access)). 
```

Comment:
- These files can be downloaded using a file manifest and the score-client

- im not sure this note is needed/adding it here is a bit confusing.  makes it sound like QC files are available for public download. [(note: all molecular files are controlled and will be available for the public to download with [DACO approval](data-access)). ]